### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'removeEventListener')

### ISSUE THREAD

**Issue by katspaugh** [wavesurfer-js/wavesurfer.js#2712]
### Discussed in https://github.com/wavesurfer-js/wavesurfer.js/discussions/2711

<div type='discussions-op-text'>

<sup>Originally posted by **4magnum** March 11, 2023</sup>

I am using the REGIONS plugin and get the following error when calling **clearAllRegions** or **remove** a single region. The issue started today and am wondering if anyone can help shed some light on this. Thanks!

region.js:455 Uncaught TypeError: Cannot read properties of undefined (reading 'removeEventListener')
    at region.js:455:28
    at wavesurfer.js:3042:12
    at Array.forEach (<anonymous>)
    at e.fireEvent (wavesurfer.js:3041:28)
    at e.value (region.js:139:18)
    at HTMLButtonElement.<anonymous> (test_video.html?gI5JKhoZbYs.mp4:1206:17)
    at HTMLButtonElement.dispatch (jquery-3.4.1.min.js:2:42571)
    at v.handle (jquery-3.4.1.min.js:2:40572)</div>

**Comment by katspaugh** [comment]
Looks like a regression from #2521.

**Comment by pokemon963** [comment]
Yes! I am facing the same issue. There seems to be an issue with the regions.js file:

```
this.on('remove', () => {
            this.contentEl.removeEventListener('blur', this.onContentBlur.bind(this));
            this.contentEl.removeEventListener('click', this.onContentClick.bind(this));
            this.removeButtonEl.removeEventListener('click', this.onRemove.bind(this));
        });
```

**Comment by katspaugh** [comment]
@4magnum @pokemon963 should be fixed in #2713

You can preview it here: https://fix-regions.wavesurfer-js.pages.dev/example/video-annotation/

Deleting a region doesn't throw for me anymore. ✅
